src/screens/LoginPage.js

```diff
@@ -1,14 +1,115 @@
-import { StyleSheet, Text, View } from 'react-native'
-import React from 'react'
+import { Pressable, StyleSheet, Text, TextInput, View } from 'react-native'
+import React, { useState } from 'react'
+
+const LoginPage = ({ navigation }) => {
+
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+
+
+  const HandleOnPress = () => {
+    console.log(email, password)
+  }
```

Comment:
```suggestion
  const HandleOnPress = () => {
    console.log({email, password})
  }
```

Bu bölümü düzeltmen gerekli